### ISSUE TITLE
Add Resource attributes to Zipkin exporter

### ISSUE THREAD

**Issue by iNikem** [open-telemetry/opentelemetry-java#1970]
[Specification says](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/sdk_exporters/zipkin.md#attribute):

> OpenTelemetry Span and Resource `Attribute`(s) MUST be reported as `tags` to Zipkin.

But `ZipkinSpanExporter` currently translates only `service.name` resource attribute and completely ignores all others.

**Comment by Oberon00** [comment]
Hmm, this specification seems a bit dubious to me. There can be a lot of resource attributes. Do we really want to duplicate them on each span.

**Comment by iNikem** [comment]
Do we have any other option? We don't want to loose them completely, do we?

**Comment by anuraaga** [comment]
I never noticed Resource there for some reason - that does seem quite problematic since I think most zipkin deployments will be destroyed if we send all the resource attributes in every span. Considering Zipkin exporter will commonly be used with Zipkin (collector will generally be used with OTLP I guess?), I think we are effectively required to have a configuration knob in the SDK to filter attributes if we do indeed add these.

https://github.com/open-telemetry/opentelemetry-specification/issues/823 was pushed to post-GA but it makes me want to remove Resource from the zipkin exporter spec until post-GA too since I think they both need to be considered together.

**Comment by anuraaga** [comment]
Sent https://github.com/open-telemetry/opentelemetry-specification/pull/1185 let's see what people think

**Comment by jkwatson** [comment]
Marking this as blocked by the spec until 1185 is resolved.

**Comment by Oberon00** [comment]
open-telemetry/opentelemetry-specification#1185  has been merged, removing resources from Zipkin for now, so I'm removing the "Bug" label since we are now conformant.

The follow-up https://github.com/open-telemetry/opentelemetry-specification/issues/1196 is only "allowed-for-ga", so I think we should not have this issue required-for-ga eiher.

**Comment by jkwatson** [comment]
I wonder if we can just close this and open a new issue if/when 1196 gets resolved in the spec?

**Comment by jkwatson** [comment]
closing due to inactivity, and no support in the specs.
